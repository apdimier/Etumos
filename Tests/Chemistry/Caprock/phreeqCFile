
PHASES

      Ankerite
          CaFe0.7Mg0.3(CO3)2 + 4H+ =  + 1.0Ca++ + 0.7Fe++ + 0.3Mg++ + 2.0H2O + 2.0CO2
                    log_k   1.2140000000e+01

SOLUTION                  1-1 caprock
   temp 8.0000000000e+01
   pH    6.5400000000e+00 
   pe    4.0000000000e+00
   units mol/l
                       Al    1.5310000000e-07
                        C    2.1800000000e-03
                       Ca    1.5280000000e-02
                       Cl    2.6010000000e-01
                       Fe    1.5340000000e-05
                        K    1.1900000000e-02
                       Mg    8.9370000000e-04
                       Na    2.5430000000e-01
                        S    1.8410000000e-02
                       Si    5.3710000000e-04

EQUILIBRIUM_PHASES        1-1 caprock
                  Calcite    0.0000000e+00  2.4982263e-01
                 Ankerite    0.0000000e+00  1.2109563e-02
               Montmor-Na    0.0000000e+00  3.4064383e-02
                Kaolinite    0.0000000e+00  5.8114397e-03
                   Illite    0.0000000e+00  2.6052812e-03
                   Quartz    0.0000000e+00  8.3231375e-02
                Anhydrite    0.0000000e+00  1.1019689e-02
                   Pyrite    0.0000000e+00  8.3347919e-03
                 Goethite    6.0000000e-04  0.0000000e+00
               Chalcedony    -2.3000000e-01  0.0000000e+00
             Dolomite-dis    -1.2400000e+00  0.0000000e+00
                 Siderite    -8.0000000e-01  0.0000000e+00
SELECTED_OUTPUT
  -high_precision true
KNOBS
  -iterations 500
  -diag true 
  -tolerance 1.e-15
SOLUTION 0
PRINT
  -reset	true
# hereafter the -cells parameter is the 
# only relevant parameter 
TRANSPORT
  -cells          0
  -time_step          100.
  -shifts          5
  -lengths        0.1
  -flow_direction diff
  -boundary_conditions        closed closed
  -print_frequency      1
  -warnings False
END
