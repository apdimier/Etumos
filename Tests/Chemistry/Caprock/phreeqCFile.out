                    


number of aqueous primary species                        29
number of aqueous secondary species                     119
number of minerals                                       12
number of sorbed species                                  0

pH                                          6.53598833729
pe                                           -2.73794794149
water activity                               0.990375551459
ionicstrength                                0.318239432701
temperature                                            80.0
electrical_balance                       -4.58847292997e-06
total H                                       111.022868346
total O                                       55.6029037926


 primary species: mol/l

                H(1) 1.0
                  Al 1.59767104798e-07
               C(-2) 0.0
               C(-3) 4.08207917928e-08
               C(-4) 3.83145293828e-15
                C(2) 2.67849349438e-15
                C(4) 0.00217362482002
                  Ca 0.0155329021309
              Cl(-1) 0.264965104831
               Cl(1) 0.0
               Cl(3) 0.0
               Cl(5) 0.0
               Cl(7) 0.0
               Fe(2) 1.52177201254e-05
               Fe(3) 1.41021068215e-13
                H(0) 4.2722952584e-10
                   K 0.0129789766549
                  Mg 0.000837085189651
                  Na 0.26345055706
                O(0) 1.15348442205e-17
               S(-2) 2.23287689548e-08
                S(2) 2.69613909902e-12
                S(3) 1.13821274381e-25
                S(4) 2.24481506031e-12
                S(5) 0.0
                S(6) 0.0212929621661
                S(7) 0.0
                S(8) 0.0
                  Si 0.000573926025152


 secondary species: mol/l

                  H+ 3.70387792544e-07
                 H2O 55.5253251008
               HAlO2 1.10244946261e-08
              NaAlO2 6.64733479618e-09
            Al(OH)2+ 1.74359870607e-10
              AlOH+2 3.40137838957e-12
                Al+3 7.80699927105e-15
              AlSO4+ 2.75498566966e-15
           Al(SO4)2- 5.6867182423e-16
Al2(OH)2+4Al3(OH)4+5Al13O4(OH)CO2 3.58886800195e-23
Al3(OH)4+5Al13O4(OH)CO2 3.22609591696e-30
       Al13O4(OH)CO2 0.0
                 CO2 0.000534566243725
              NaHCO3 9.10408855225e-05
             CaHCO3+ 9.00382659768e-05
               CaCO3 7.12808556577e-06
             MgHCO3+ 3.46895556033e-06
             FeHCO3+ 2.15421575976e-06
               CO3-2 1.32742552076e-06
               MgCO3 7.8100526059e-08
              NaCO3- 3.91761990279e-08
               FeCO3 2.40695558852e-08
              FeCO3+ 1.78958413357e-19
               CaSO4 0.0021725089683
               CaCl+ 0.000265452013584
             CaHCO3+ 9.00382659768e-05
               CaCl2 2.63920602632e-05
               CaCO3 7.12808556577e-06
               CaOH+ 2.37329182598e-09
                NaCl 0.00764052949199
               CaCl+ 0.000265452013584
                 KCl 0.000114809854011
               MgCl+ 3.39285826779e-05
               CaCl2 2.63920602632e-05
               FeCl+ 8.14121952534e-07
                 HCl 1.04763132289e-08
               FeCl2 7.42198979472e-10
             FeCl4-2 6.80187636685e-10
              FeCl2+ 1.0091600255e-20
              FeCl+2 3.37875772809e-21
              FeCl4- 3.31763348199e-25
                ClO- 0.0
               HClO2 0.0
             FeHCO3+ 2.15421575976e-06
               FeSO4 1.19822048995e-06
               FeCl+ 8.14121952534e-07
               FeCO3 2.40695558852e-08
               FeOH+ 4.51431730899e-09
               FeCl2 7.42198979472e-10
             FeCl4-2 6.80187636685e-10
             Fe(OH)2 8.26743656256e-14
            Fe(OH)3- 1.65260315764e-17
           Fe(OH)4-2 1.99458273217e-25
            Fe(OH)2+ 6.77123940232e-14
              FeOH+2 1.95493985438e-16
            Fe(OH)4- 9.2096826407e-17
              FeCO3+ 1.78958413357e-19
                Fe+3 2.10959376997e-20
              FeCl2+ 1.0091600255e-20
              FeSO4+ 7.40330895549e-21
              FeCl+2 3.37875772809e-21
           Fe(SO4)2- 3.16429092989e-23
              FeCl4- 3.31763348199e-25
Fe2(OH)2+4Fe3(OH)4+5KSO4- 1.44074418759e-29
     Fe3(OH)4+5KSO4- 2.42170568768e-39
               KSO4- 0.000379569158461
                 KCl 0.000114809854011
               KHSO4 4.59806594558e-10
                 KOH 9.48213080219e-11
                Mg+2 0.000395804433036
               MgCl+ 3.39285826779e-05
             MgHCO3+ 3.46895556033e-06
               MgCO3 7.8100526059e-08
      Mg4(OH)4+4NaCl 1.93373760131e-27
                NaCl 0.00764052949199
              NaSO4- 0.00441504410556
              NaHCO3 9.10408855225e-05
             NaHSiO3 3.75840916178e-06
              NaCO3- 3.91761990279e-08
                NaOH 2.77776693203e-08
              NaAlO2 6.64733479618e-09
                S3-2 3.36732747478e-22
                S4-2 6.48564149459e-27
              S4O6-2 4.19236519398e-30
                S5-2 8.06421587101e-32
              S3O6-2 1.30933160979e-37
              S5O6-2 0.0
                  O2 0.0
                 H2S 8.91038077172e-09
                 S-2 3.93101639928e-13
                S2-2 1.15341707355e-17
                S3-2 3.36732747478e-22
                S4-2 6.48564149459e-27
                S5-2 8.06421587101e-32
              HS2O3- 1.14216373058e-18
               SO3-2 4.50270413359e-13
                 SO2 1.1194320463e-16
               H2SO3 1.29847561415e-17
              S2O5-2 7.8130901247e-29
              S4O6-2 4.19236519398e-30
              S3O6-2 1.30933160979e-37
              S5O6-2 0.0
              NaSO4- 0.00441504410556
               CaSO4 0.0021725089683
               MgSO4 0.000403805117851
               KSO4- 0.000379569158461
               FeSO4 1.19822048995e-06
               HSO4- 5.85514898781e-07
               KHSO4 4.59806594558e-10
              AlSO4+ 2.75498566966e-15
           Al(SO4)2- 5.6867182423e-16
               H2SO4 2.14716634932e-17
              FeSO4+ 7.40330895549e-21
           Fe(SO4)2- 3.16429092989e-23
             NaHSiO3 3.75840916178e-06
              HSiO3- 1.57337888514e-06
            H2SiO4-2 3.77818862921e-13
H6(H2SiO4)H4(H2SiO4)4-4 1.36927430977e-13
       H4(H2SiO4)4-4 1.28247246361e-20


 sorbed species: moles



     SI( target saturation index):        SI < 0 undersaturation, 0: equilibrium and supersaturation otherwise





                 mineral species:            SI                   moles

                     Anhydrite          0.00000000e+00       7.35161660e-03

                      Ankerite         -1.02195949e-02       0.00000000e+00

                       Calcite         -8.88178420e-16       2.57177472e-01

                    Chalcedony         -2.30000000e-01       1.15932592e-01

                  Dolomite-dis         -1.24000000e+00       8.45569113e-03

                      Goethite          6.00000000e-04       7.91204578e-03

                        Illite          3.55271368e-15       1.17834239e-03

                     Kaolinite         -4.44089210e-16       1.85674711e-02

                    Montmor-Na         -1.77635684e-15       2.07529414e-02

                        Pyrite          1.06581410e-14       8.89984932e-03

                        Quartz         -1.65246774e-03       0.00000000e+00

                      Siderite         -8.50658459e-01       0.00000000e+00

